docs/DeveloperGuide.md

```diff
@@ -1000,3 +1000,24 @@ testers are expected to do more *exploratory* testing.
 similar to the test cases found in `Editing Seller Details`.
 
 </box>
+
+### B.2 Add Seller/Buyer command
+
+#### B.2.1 Motivation
+- The `addSeller` and `addBuyer` commands currently identify individuals within the `Persons` list using `name` as a unique identifier. There are two main reasons for this approach.
+- Firstly, we have been using the `name` to identify `Buyers` and `Sellers` in various commands, such as adding or deleting a house. 
+- Secondly, to distinguish between individuals sharing the same `name`, we append numbers to their names, resulting in unique identifiers like `John Doe 1` and `John Doe 2`, which is why we allow alphanumeric characters in the `name` field.
```

Comment:
` we append numbers to their names, resulting in unique identifiers` 
I think instead of saying we append, we should say user can append numbers, cuz i think our code implementation does not have this.

Reply:
Okay I will make this change.